apps/cow-fi/data/home/const.tsx

```diff
@@ -96,3 +111,56 @@ export const CHANNEL_LIST = [
     iconImage: IMG_SNAPSHOT,
   },
 ]
+
+export const PRODUCT_CONTAINERS = (
```

Comment:
not sure if const is the place for this. Now it worked to not create a component cause you were able to build it from static information (no props needed), but not always be possible

I will approve, just to have it DRYer the code, but I think we will need some other general solution imo